README.md

```diff
@@ -47,14 +47,14 @@ making it much easier to get a running index server.
 
 Table of Contents
 
-- [**pypiserver - minimal PyPI server for use with pip/easy_install**](#pypiserver---minimal-pypi-server-for-use-with-pipeasy_install)
+- [**pypiserver - minimal PyPI server for use with pip/easy\_install**](#pypiserver---minimal-pypi-server-for-use-with-pipeasy_install)
```

Comment:
As we use the standard `mdformat` configuration, it requires `easy_install` without a `_`. Do you mind adjusting?

We should probably fix this in the formatter's config some day!

Reply:
@dee-me-tree-or-love I see that you pushed up a commit to fix those. Thanks!